### DISCUSSION TITLE
HACS !

### DISCUSSION THREAD

**Discussion by gcoan** [BJReplay/ha-solcast-solar#147] [General]
🎉

Upgraded to HACS 2.0 and see that Solcast has made it !!!

![image](https://github.com/user-attachments/assets/eed2b4ba-0a39-41cf-82af-86b9a0356f24)

**Comment by BJReplay** [comment]
It's listed there because you've installed it, but it's not yet available to download as a **default** integration within HACS.

The update behaviour is smoother in HACS 2.0 (it's the same as if you had experimental turned on in HACS 1.3.4) - you'll get a notification on settings that there's an upgrade.

**Reply by gcoan** [reply]
Damn. I thought it was now there on HACS, but of course I have Solcast as a custom repository

**Comment by BJReplay** [comment]
Closing - for the moment, anyway - probably 10 weeks away until this appears in HACS default, based on my guestimate of the number of PRs getting closed at the moment.
